packages/ui/src/components/shadcn/ui/command.tsx

```diff
@@ -41,20 +41,46 @@ const CommandInput = React.forwardRef<
   React.ElementRef<typeof CommandPrimitive.Input>,
   React.ComponentPropsWithoutRef<typeof CommandPrimitive.Input> & {
     wrapperClassName?: string
+    showCloseIcon?: boolean
+    showSearchIcon?: boolean
+    handleClose?: () => void
   }
->(({ className, wrapperClassName, ...props }, ref) => (
-  <div className={cn('flex items-center border-b px-3', wrapperClassName)} cmdk-input-wrapper="">
-    <Search className="h-4 w-4 shrink-0 opacity-50" />
-    <CommandPrimitive.Input
-      ref={ref}
-      className={cn(
-        'flex h-9 w-full rounded-md bg-transparent py-3 text-xs text outline-none placeholder:text-muted disabled:cursor-not-allowed disabled:opacity-50 border-none focus:ring-0',
-        className
+>(
+  (
+    {
+      className,
+      wrapperClassName,
+      showCloseIcon = false,
+      showSearchIcon = true,
+      handleClose,
+      ...props
+    },
+    ref
+  ) => (
+    <div className={cn('flex items-center border-b px-3', wrapperClassName)} cmdk-input-wrapper="">
+      {showSearchIcon && <Search className="h-4 w-4 shrink-0 opacity-50" />}
+      <CommandPrimitive.Input
+        ref={ref}
+        className={cn(
+          'flex h-9 w-full rounded-md bg-transparent py-3 text-xs text outline-none placeholder:text-muted disabled:cursor-not-allowed disabled:opacity-50 border-none focus:ring-0',
+          className
+        )}
+        {...props}
+      />
+      {showCloseIcon && (
```

Comment:
What's this for ?

Reply:
It's this x to reset the input. Wrong naming now that you mention it...

https://github.com/user-attachments/assets/9b8c748d-868a-4a7a-ac1b-089d01025117

Reply:
`showResetIcon` now